charts/trento-server/charts/trento-wanda/templates/deployment.yaml

```diff
@@ -29,6 +29,30 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       initContainers:
+        - name: create-wanda-db
+          image: "{{ .Values.postgresql.image.registry }}/{{ .Values.postgresql.image.repository }}:{{ .Values.postgresql.image.tag }}"
+          command:
+            - sh
```

Comment:
I would be nice to explore if this piece of code could be "templated", as it is repeated 3 times.
Anyway, not a biggie